src/main/java/com/cloudbees/jenkins/support/impl/ProcFilesRetriever.java

```diff
@@ -55,7 +53,7 @@ protected static String getNodeName(Node node) {
 
     @Override
     public void addContents(@NonNull Container container) {
-        Jenkins j = Helper.getActiveInstance();
+        Jenkins j = Jenkins.getActiveInstance();
```

Comment:
:NIT: should be part of a different PR (I assume it's safe to use and should just get rid of the helper version and just use the one in the `Jenkins` class.

Reply:
+1, and possibly more than just NIT because https://github.com/jenkinsci/jenkins/pull/3195... :)

Reply:
It may be deprecated in later baselines, but it works as advertised and will continue to.